lib/armadietto.js

```diff
@@ -139,63 +139,55 @@ class Armadietto {
 
   async dispatch (req, res) {
     const method = req.method.toUpperCase();
-    const uri = url.parse(req.url, true);
+    let pathname = new url.URL(req.url, `http://${this._options.http.host}${req.url}`).pathname;
```

Comment:
This is fine. 

As [the docs](https://nodejs.org/docs/latest/api/url.html#new-urlinput-base) note "URL class is also available on the global object". So, when we do future work, we can use `new URL(...`

Reply:
Ah yes, not sure why I didn't think of that. I've gone ahead and switched it per your comment, as it also means we don't have to `require("url")`